File: js_project/Arrays/ArrayElementInitializer.js
let a = [];
for (let i =0; i<10; i++){
    let value = (i+1)*5;
    a.push(value);

}
//console.log(a);
let reverse =[];
for (let i= a.length-1; i>=0; i--){
    reverse.push(a[i]);
}
console.log(reverse.join(" "))

File: java_project/Array/ArrayElementInitializer.java
package Array;

public class ArrayElementInitializer {
    public static void main(String[] args) {
        int a[]= new int [10];
        for (int i=0; i<a.length; i++){
            a[i]=(i+1)*5;
            System.out.print(a[i]+"\t");
        }
        System.out.println("--the correct order");
         for (int i=a.length-1; i>=0; i--){
           // System.out.println("The reverse order ");
            System.out.print(a[i]+"\t");
        }
        System.out.println("--The reverse order "); 
    }
}
